src/components/common/post/section/PostSection.jsx

```diff
@@ -0,0 +1,20 @@
+import s from './PostSection.module.css';
+import PostSectionChild from "./child/PostSectionChild.jsx";
+
+/* eslint-disable react/prop-types */
+function PostSection(props) {
```

Comment:
Just FYI you can also do it like this - `function PostSection({ section }) {` - deconstucting props